code/modules/mob/living/simple_animal/hostile/faithless.dm

```diff
@@ -1,6 +1,6 @@
 /mob/living/simple_animal/hostile/faithless
 	name = "The Faithless"
-	desc = "The Wish Granter's faith in humanity, incarnate"
+	desc = "A being of shadow. It posesses a menacing gaze and a tortured voice."
```

Comment:
OK this one was probably iconic too, but to be fair yours is better in this case